### PR TITLE
added genres api-blueprint

### PR DESCRIPTION
**Pull request by robyncuppen**

### CODE REVIEW

File: blueprint/media/genres.mson [hunk 1]
```diff
@@ -0,0 +1,11 @@
+## genresAttributes (object)
++ name: Action (string)
```
Comment (wopian) [inline blueprint/media/genres.mson]: ❌ `string` not needed

File: blueprint/media/genres.mson [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
++ slug: action (string)
++ description (string)
```
Comment (wopian) [inline blueprint/media/genres.mson]: ❌ `description (nullable)`

**Review by wopian — CHANGES_REQUESTED** [review]

**Comment by robyncuppen** [comment]
should I still change this as I just found out that #7 says that genre is deprecated?

**Comment by wopian** [comment]
>should I still change this as I just found out that #7 says that genre is deprecated?

I'd still document it. Its still returning results & is used by quite a few API users.

**Review by wopian — APPROVED** [review]

**Pull request merged**